### PR TITLE
DT-1167 Use latest versoins (mitigates CVE-2020-5421)

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -32,21 +32,21 @@ dependencies {
...
+  implementation("org.elasticsearch.client:elasticsearch-rest-high-level-client:7.9.2")
+  implementation("org.elasticsearch.client:elasticsearch-rest-client:7.9.2")
 
-  implementation("com.amazonaws:aws-java-sdk-core:1.11.854")
+  implementation("com.amazonaws:aws-java-sdk-core:1.11.873")
 
-  implementation("io.springfox:springfox-swagger2:2.9.2")
-  implementation("io.springfox:springfox-swagger-ui:2.9.2")
-  implementation("io.springfox:springfox-bean-validators:2.9.2")
+  implementation("io.springfox:springfox-swagger2:3.0.0")
```
Comment (andymarke) [inline build.gradle.kts:41]: Does this work?

**Review by andymarke — APPROVED** [review]
LGTM

File: build.gradle.kts (hunk 1 above)
Comment (petergphillips) [reply build.gradle.kts:41]: didn't do when I tried it on other projects

File: build.gradle.kts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.0.2"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.0.6"
   kotlin("plugin.spring") version "1.4.0"
```
Comment (petergphillips) [inline build.gradle.kts:3]: kotlin bump

File: build.gradle.kts (hunk 1 above)
Comment (petergphillips) [reply build.gradle.kts:41]: would be good to investigate though :grin:

Comment (mikehalmamoj) [reply build.gradle.kts]: I've updated to `io.springfox:springfox-boot-starter:3.0.0` instead after copying @andrewrlee and that worked fine.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**